consensus/Cargo.toml

```diff
@@ -17,6 +17,7 @@ num-derive = { version = "0.2.5", default-features = false }
 num-traits = { version = "0.2.8", default-features = false }
 parity-multiaddr = { version = "0.5.0", default-features = false }
 protobuf = "~2.7"
```

Comment:
do we still need this?

Reply:
Yes, i have another PR that will remove this :)